corehq/apps/sso/tests/test_request_helpers.py

```diff
@@ -0,0 +1,238 @@
+from testil import eq
+from django.test import TestCase, RequestFactory, override_settings
+
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.domain.models import Domain
+from corehq.apps.sso.exceptions import SingleSignOnError
+from corehq.apps.sso.models import (
+    AuthenticatedEmailDomain,
+    IdentityProvider,
+    TrustedIdentityProvider,
+)
+from corehq.apps.sso.utils.request_helpers import (
+    get_request_data,
+    is_request_using_sso,
+    is_request_blocked_from_viewing_domain_due_to_sso,
+)
+from corehq.apps.users.models import WebUser
+from corehq.apps.sso.tests import generator
+
+
+@override_settings(SAML2_DEBUG=False)
+def test_get_request_data():
```

Comment:
this was a suggestion from @millerdev in the past whenever there were simple tests that didn't require any database setup, but I guess that would fall under `SimpleTestCase` too. Daniel, do you have any opinions here? ^_^

Reply:
I like function tests because they require less boilerplate than class-based tests. Don't think `SimpleTestCase` provides any distinct advantages unless you need some of its assert methods (aside: it's kinda sad that Python doesn't provide functional versions of all of those assert methods since there is no particular reason why they need to be tied to a class instance). In the mean time `testil` provides the most common ones, with snake-case to boot.